pkg/apis/storage/validation/validation.go

```diff
@@ -403,7 +403,8 @@ func ValidateCSIDriver(csiDriver *storage.CSIDriver) field.ErrorList {
 
 // ValidateCSIDriverUpdate validates a CSIDriver.
 func ValidateCSIDriverUpdate(new, old *storage.CSIDriver) field.ErrorList {
-	allErrs := ValidateCSIDriver(new)
+	allErrs := field.ErrorList{}
+	allErrs = append(allErrs, apivalidation.ValidateCSIDriverName(new.Name, field.NewPath("name"))...)
```

Comment:
Is this `Metadata.Name`?  You shouldn't be validating name on update, you should be calling object metadata to validate that?  Name is always immutable in every object in Kubernetes.

Reply:
I've adjusted this to use `apivalidation.ValidateObjectMetaUpdate`.